.github/workflows/patch-asyncio.ps1

```diff
@@ -0,0 +1,3 @@
+$fileName = 'c:\hostedtoolcache\windows\python\3.8.5\x64\lib\asyncio\__init__.py'
```

Comment:
This patch looks like a bit of a hack. It will only work with this specific version of python. When the GHA machine image is updated to a newer version it will break. Please document this hack a little better, is there a Python issue to reference or something?

Reply:
Yeah, it's admittedly pretty hacky. The issue comes from the Tornado web server that Jupyter relies on.  The issue is documented at https://github.com/tornadoweb/tornado/issues/2608.  It's still current as seen at https://www.tornadoweb.org/en/stable/releases/v6.0.4.html#general-changes.  There's an upstream issue in the asyncio library that can be seen at https://bugs.python.org/issue37373.

To address this patch breaking when Python gets upgraded, I can apply the patch in a virtualenv instead which wouldn't require a hard path with a version in it. Would that work?

Reply:
You could obtain the path with something like `python -c "import asyncio; print(asyncio.__file__)"`, but then it may be easier to write the whole script in python.

>To address this patch breaking when Python gets upgraded, I can apply the patch in a virtualenv instead which wouldn't require a hard path with a version in it. Would that work?

That could work two, but I does not look like a virtual env is being used yet.